src/library/importer.py

```diff
@@ -1,17 +1,20 @@
 from django_tenants.utils import schema_context
+from library.utils import get_library_schema_name
 from quest_manager.admin import QuestResource
 from quest_manager.models import Quest
 
-from .utils import library_schema_context
 
-
-def import_quests_to(*, destination_schema, quest_import_ids):
+def import_quests_to(*, destination_schema, quest_import_ids, source_schema=None):
     """
     :param destination_schema: schema to import the quest to
     :param quest_import_ids: list of quest import ids
+    :param source_schema: Optional. The source where to import the quest import ids
```

Comment:
What happens if this isn't provided?

Reply:
Looks like it defaults to the library, what is happening if it's not the library?  Can this be used to import directly from one deck to another?  I think the missing docstring is the problem.  Don't know what this function is doing and when.

If library is default, can we put source_schema=get_library_schema_name in the function parameters directly? (not sure if we can provide a function or not...